en/appbuilder/generate-app/generate-app-overview.md

```diff
@@ -18,12 +18,12 @@ _keywords: App Builder, Web App Builder, Design Systems, Design Systems UX, UI k
     </div>
 </section>
 
-Previewing generated code and downloading your work as an Angular app is the final step after designing one application. The design and development user story will be completed once the application is generated with all styiling and layout properties. Currently, Indigo.Design App Builder offers options to upload all your application files to a Github repository or download them as a package and then run it locally. 
```

Comment:
**I'd embrace bullets whenever we can. Also, switch to active voice. For example:**

You can generate code for any application designed in Indigo.Design app builder. This allows you continue developing your app in your preferred IDE (e.g., Visual Studio Code).

You can view the generated code (HTML, TS, CSS) for each **view** by toggling `code-view ON` when previewing the app. But the `generate app` workflow also includes all the required files, and source code, for you to run your app as a stand-alone Angular application.

You can generate the source code for your designed app in two ways:

1. Publish your app to a GitHub repository
2. Download generated files as a ZIP

Any of the two approaches will let you compile and run your application in your IDE.